core/store/migrate/migrations/0125_keeper_flatten_schema_version.sql

```diff
@@ -0,0 +1,55 @@
+-- +goose Up
+UPDATE pipeline_specs
+SET dot_dag_source = 'encode_check_upkeep_tx   [type=ethabiencode
```

Comment:
Why do we change the dot_dag_source, I remember we set it to be the empty string last time? Isn't it sufficient if we just change the schema version to 1 and keep the dot_dag_source empty?